test/integration/config.yml

```diff
@@ -27,8 +27,9 @@
     es_heap_size: 1g
     #add a custom log4j file
     es_config_log4j2: "./files/logging/log4j2.properties.custom.j2"
+    es_config_jvmoptionsj2: "./files/jvm/jvm.options.custom.j2"
```

Comment:
Since this template is being used for testing and is just a copy of `templates/jvm.options.j2` could you please move it to `test/integration/config/templates/jvm.options.custom.j2`